{"name": "Test create issue with correct data", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestIssues' object has no attribute '__qualname__'", "trace": "self = <src.tests.test_selenium.TestIssues object at 0x000002ED51445198>\nissue_data = {'description': 'this is a test Story', 'project': 'AQAPython (AQAPYTHON)', 'summary': 'AlexART - Test Story', 'type': 'Story'}\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"44cb88d6db162b49bba66dd028d5ee4c\")>\njira_rest = <src.rest.jira.Jira object at 0x000002ED51423DA0>\n\n    @allure.step\n    @allure.tag('UI')\n    @allure.feature('Issues management')\n    @allure.story('Create issue')\n    @allure.title(\"Test create issue with correct data\")\n    @allure.severity(allure.severity_level.CRITICAL)\n    @pytest.mark.parametrize(\"issue_data\", [{\"project\": \"AQAPython (AQAPYTHON)\", \"summary\": \"AlexART - Test Story\", \"type\": \"Story\",\n                                             \"description\": \"this is a test Story\"},\n                                            {\"project\": \"AQAPython (AQAPYTHON)\", \"summary\": \"AlexART - Test Bug\", \"type\": \"Bug\",\n                                             \"description\": \"this is a test Bug\", \"priority\": \"Medium\"}])\n    def test_create_issue_positive(self, issue_data, driver, jira_rest):\n        # login\n        login_page = LoginPage(driver)\n        login_page.go(\"http://jira.hillel.it:8080/\")\n        login_page.login(\"Alexander_Artemov\", \"Alexander_Artemov\")\n    \n        # create issue\n        dashboard_page = GeneralPage(driver)\n        issue_key = dashboard_page.create_issue(**issue_data)\n    \n>       allure.attach(self.__qualname__ + 'screenshot', self.driver.get_screenshot_as_png(), type=AttachmentType.PNG)\nE       AttributeError: 'TestIssues' object has no attribute '__qualname__'\n\nsrc\\tests\\test_selenium.py:133: AttributeError"}, "steps": [{"name": "test_create_issue_positive", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestIssues' object has no attribute '__qualname__'\n", "trace": "  File \"c:\\users\\aartemov\\pycharmprojects\\pythoncourse\\phase4\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 141, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\aartemov\\PycharmProjects\\pythoncourse\\phase5\\src\\tests\\test_selenium.py\", line 133, in test_create_issue_positive\n    allure.attach(self.__qualname__ + 'screenshot', self.driver.get_screenshot_as_png(), type=AttachmentType.PNG)\n"}, "parameters": [{"name": "issue_data", "value": "{'project': 'AQAPython (AQAPYTHON)', 'summary': 'AlexART - Test Story', 'type': 'Story', 'description': 'this is a test Story'}"}, {"name": "driver", "value": "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"44cb88d6db162b49bba66dd028d5ee4c\")>"}, {"name": "jira_rest", "value": "<src.rest.jira.Jira object at 0x000002ED51423DA0>"}], "start": 1537453383364, "stop": 1537453418327}], "parameters": [{"name": "issue_data", "value": "{'project': 'AQAPython (AQAPYTHON)', 'summary': 'AlexART - Test Story', 'type': 'Story', 'description': 'this is a test Story'}"}], "start": 1537453383364, "stop": 1537453418328, "uuid": "72f739e6-66b9-4ddd-89d8-00cbd86ecd42", "historyId": "372fb0944749165711077e972274655b", "fullName": "src.tests.test_selenium.TestIssues#test_create_issue_positive[issue_data0]", "labels": [{"name": "story", "value": "Create issue"}, {"name": "severity", "value": "critical"}, {"name": "feature", "value": "Issues management"}, {"name": "tag", "value": "UI"}, {"name": "parentSuite", "value": "src.tests"}, {"name": "suite", "value": "test_selenium"}, {"name": "subSuite", "value": "TestIssues"}, {"name": "host", "value": "odp1210013"}, {"name": "thread", "value": "22744-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.tests.test_selenium"}]}